electricitymap/contrib/config/capacity.py

```diff
@@ -1,4 +1,25 @@
 from datetime import datetime
+from pathlib import Path
+
+from electricitymap.contrib.config.reading import read_zones_config
+
+CONFIG_DIR = Path(__file__).parent.parent.parent.parent.joinpath("config").resolve()
+
+ZONES_CONFIG = read_zones_config(CONFIG_DIR)
+# Get productionCapacity source to zones mapping
+CAPACITY_PARSER_SOURCE_TO_ZONES = {}
+for zone_id, zone_config in ZONES_CONFIG.items():
+    if zone_config.get("parsers", {}).get("productionCapacity") is None:
+        continue
+    source = zone_config.get("parsers", {}).get("productionCapacity").split(".")[0]
+    if source not in CAPACITY_PARSER_SOURCE_TO_ZONES:
+        CAPACITY_PARSER_SOURCE_TO_ZONES[source] = []
+    CAPACITY_PARSER_SOURCE_TO_ZONES[source].append(zone_id)
+
+ZONE_TO_CAPACITY_PARSER_SOURCE = {}
```

Comment:
I think it would be better performance wise to do it with list comprehension.
```suggestion
ZONE_TO_CAPACITY_PARSER_SOURCE = {
    zone: source
    for source, zones in ZONES_CONFIG.items()
    for zone in zones
}
```